docs/docs/mdx/plugins.md

```diff
@@ -0,0 +1,87 @@
+---
+title: MDX plugins
+---
+
+## Table of contents
+
+- [Gatsby remark plugins](#gatsby-remark-plugins)
+- [remark plugins](#remark-plugins)
+
+## Gatsby remark plugins
+
+gatsby-mdx is compatible with all of the [gatsby-remark
+plugins](https://next.gatsbyjs.org/packages/gatsby-remark-images/?=gatsby-remark),
```

Comment:
Will these URLs need to be updated at some point?

Reply:
Yes :)

```suggestion
plugins](/packages/gatsby-remark-images/?=gatsby-remark),
```